tests/integration-tests/tests/ad_integration/test_ad_integration/test_ad_integration/pcluster.config.update.yaml

```diff
@@ -42,7 +42,8 @@ DirectoryService:
   DomainAddr: {{ ldaps_uri }}
   PasswordSecretArn: {{ password_secret_arn }}
   DomainReadOnlyUser: {{ ldap_default_bind_dn }}
-  LdapTlsReqCert: never
+  LdapTlsCaCert: {{ ldap_tls_ca_cert }}
+  # LdapTlsReqCert: never
```

Comment:
I think the certificate verification should be parameterizable because we publicly state that we support SimpleAD and MsAD over LDAP, LDAPS and LDAPS without certificate verification.

Reply:
Are you suggesting we should run three tests to create three clusters to test all three scenarios?

Reply:
I totally agree with you. Here we are testing the case when `LdapTlsReqCert` is not specified. Can we leave the further improvement to the future.